lib/pbench/agent/constants.py

```diff
@@ -21,6 +21,8 @@
 tm_channel_suffix_from_tms = "from-tms"
 # Channel suffix for the Tool Meister logging channel
 tm_channel_suffix_to_logging = "to-logging"
+# Tool-Meisters info key
+tms_tool_info = "tms_tool_info"
```

Comment:
Can we consider some name changes here?

Perhaps one of:

 * `tm_data_key = "tm-data-key"`
 * `tm_data_key = "tool-meister-data-key"`
 * `tool_meister_data_key = "tool-meister-data-key"`